modules/vulnerability-management-reporting20.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_module-type: CONCEPT
+[id="vulnerability-management-reporting20_{context}"]
+= Reporting vulnerabilities
+
+[role="_abstract"]
+
+Vulnerability reporting in Vulnerability Management 2.0 provides the ability to create and download an on-demand image vulnerability report that includes detected vulnerabilities and associated details. You can share this report with auditors or internal stakeholders by scheduling emails in {product-title-short} or by downloading the report and sharing it by using other methods.
```

Comment:
`workload CVEs` is preferred over `image vulnerabilities` since the launch of VM2.0.